### PR TITLE
Allow send_email method to accept email_reply_to_id

### PR DESCRIPTION
**Pull request by richardc0**
Updates the the client to allow the client to pass in an email_reply_to_id when sending a notification.

- Added files to Solution to be consistent with other clients
- Added Integration Tests to Client to test the new functionality
- Updated Unit tests to test for email_reply_to_id\n-Updated Client Code

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,4 +5,8 @@
 # The packages folder can be ignored because of Package Restore
 **/packages/*
 bin/
-obj/
\ No newline at end of file
+obj/
+.DS_Store
+Notify.userprefs
+NotifyTests/.DS_Store
+environment.sh
```
Comment (kentsanggds) [inline .gitignore]: Could group these ignored files and create headings like in other Notify repos

File: README.md [hunk 2]
```diff
@@ -174,7 +174,7 @@ Otherwise the client will raise a `Notify.Exceptions.NotifyClientException`:
 ### Email
 
 ```csharp
-EmailNotificationResponse response = client.SendEmail(emailAddress, templateId, personalisation, reference);
+EmailNotificationResponse response = client.SendEmail(emailAddress, templateId, personalisation, reference, replyToId);
```
Comment (kentsanggds) [inline README.md]: `replyToId` should probably be `emailReplyToId` for consistency

File: NotifyTests/UnitTests/NotificationUnitClientTests.cs [hunk 3]
```diff
@@ -448,5 +448,46 @@ void mockRequest(String content)
...
+			JObject expected = new JObject
+			{
+				{ "email_address", Constants.fakeEmail },
+				{ "template_id", Constants.fakeTemplateId },
+				{ "personalisation", JObject.FromObject(personalisation) },
+				{ "reference", Constants.fakeNotificationReference },
+                { "email_reply_to_id", Constants.fakeReplyToId}
+			};
+
+			mockRequest(Constants.fakeTemplatePreviewResponseJson,
```
Comment (kentsanggds) [inline NotifyTests/UnitTests/NotificationUnitClientTests.cs]: What does `Constants.fakeTemplatePreviewResponseJson` do here, is it needed? I've also seen it in other tests so might have to update those if they are not using it properly

File: .gitignore (hunk 1 above)
Comment (richardc0) [reply .gitignore]: Done

File: README.md (hunk 2 above)
Comment (richardc0) [reply README.md]: Done

**Pull request closed without merging**